src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(health, name) {
+    this.health = 100;
```

Comment:
we don't receive prop health via constructor, so you can declare it directly after class Animal and just once